app/views/state_file/landing_page/edit.html.erb

```diff
@@ -1,4 +1,4 @@
-<% state_image_path = "partner-logos/#{@state_code}gov-logo.svg" %>
+<% state_image_path = @state_code == 'md' ? "partner-logos/mdgov-logo.png" : "partner-logos/#{@state_code}gov-logo.svg" %>
```

Comment:
[dust] this makes me sad. in the future let's migrate this filename to the StateInformationService.